app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
```

Comment:
Place each argument on a new line including `self` in the `__init__` for better code style and readability. There should be no arguments on the same line with `(` and `)`